projects.js

```diff
@@ -0,0 +1,20 @@
+const users = [
```

Comment:
These are just dummy PROJECTS, not users, right?

Reply:
Good catch. Thanks, man